admin/config.sh

```diff
@@ -10,3 +10,9 @@ BACKUP_USER=listenbrainz
 BACKUP_GROUP=listenbrainz
 BACKUP_DIR_MODE=700
 BACKUP_FILE_MODE=600
+
+# rsync to FTP server configuration
+RSYNC_FULLEXPORT_HOST='10.2.2.28'
```

Comment:
Hard coded IP? We can't do that -- we need to get IP addresses from consul, so this file will also need to be generated by consul.

Reply:
Alright, will fix.